### ISSUE TITLE
Ncspot built with cover feature failed to start

### ISSUE THREAD

**Issue by yelaco** [hrkfdn/ncspot#1528]
**Describe the bug**
Hi, I tried installing and building (`--release`) ncspot with cover feature. But they both failed to start the program after authentication.

**To Reproduce**
1. Setup
- Install: `cargo install --locked ncspot --features cover`
- Build: `cargo build --release --features cover`
2. Run: 
`ncspot` 
`./target/release/ncspot`

**System (please complete the following information):**
 - OS: Ubuntu 22.04.5 LTS x86_64
 - Terminal: Alacritty
 - Version: 1.1.2
 - Installed from: cargo

**Backtrace/Debug log**
Please attach a debug log and backtrace if ncspot has crashed.

[backtrace.log](https://github.com/user-attachments/files/17326227/backtrace.log)

![image](https://github.com/user-attachments/assets/542e84ff-c110-467c-b623-4641b2e1c4af)

**Additional context**
The installation/build without cover feature works fine
The build with cover feature and no `--release` tag works fine (`cargo build --features cover`)

**Comment by hrkfdn** [comment]
This is the log. The backtrace should be in the cache folder (~/.cache/ncspot) :)

**Comment by yelaco** [comment]
:> here you go.

[backtrace.log](https://github.com/user-attachments/files/17326150/backtrace.log)

**Comment by moabeat-berlin** [comment]
Seems to be incorrectly initialized variables. This is taken from `ui/cover.rs`.

```
        let (rows, cols, mut xpixels, mut ypixels) = unsafe {
            let query: (u16, u16, u16, u16) = (0, 0, 0, 0);
            ioctl(1, TIOCGWINSZ, &query);
            query
        };
```

Since `rows` and `cols` are initialized with 0, it leads to the divide by zero in the backtrace, if they are not changed.

```
        let font_size = Vec2::new((xpixels / cols) as usize, (ypixels / rows) as usize);
```

Maybe simply initializing them with 1 would be just fine?

**Comment by HurricanePootis** [comment]
I made a pull request that should hopefully fix this

**Comment by hrkfdn** [comment]
Fixed with #1542 + released via ncspot 1.2.1
